app/models/merchant.rb

```diff
@@ -1,6 +1,9 @@
 class Merchant <ApplicationRecord
+  has_many :merchant_employees
+  has_many :users, through: :merchant_employees
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :orders, through: :item_orders
```

Comment:
A merchant can already access orders.  It goes through it’s items that have order then can just be done with a group by on order id